C2CRIBuildDir/projects/C2C-RI/src/RI_Logging/src/org/fhwa/c2cri/logger/RILogging.java

```diff
@@ -140,31 +148,31 @@ public void configureLogging(String logFileName, String configFileName, String l
             DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
             Date date = new Date();
 
-            logFile = logFileName + "." + dateFormat.format(date) + ".xml";
-
-            log = Logger.getLogger("net.sf.jameleon");
+	LoggerContext ctx = (LoggerContext) LogManager.getContext(false);
+	logFile = logFileName + "." + dateFormat.format(date) + ".xml";
+        RIXMLLayout oLayout = RIXMLLayout.createLayout();
+        oLayout.m_bUseCdata = false;
+	riAppender = FileAppender.newBuilder().withFileName(logFile).setName("riAppender").setImmediateFlush(true).setBufferedIo(true).setLayout(oLayout).build();
+	riAppender.start();
+        Logger oRoot = (Logger)LogManager.getRootLogger();
+        oRoot.addAppender(riAppender);
+        oRoot.addAppender(riGUIAppender);
+			
+
+            log = (Logger)LogManager.getLogger("C2CRIDebug");
```

Comment:
Not sure why we replace `org.fhwa.c2cri` and `net.sf.jameleon` with `C2CRIDebug`?

Reply:
I explained this some in a comment above. The log and log2 variables weren't used very much so I consolidated them into one Logger which was basically only being used for debug messages. The "org.fhwa.c2cri" in the previous log4j configuration was for the debug file, so here I am accessing the debug logger defined in the new log4j configuration